src/rgw/rgw_rest_user.cc

```diff
@@ -47,8 +48,11 @@ void RGWOp_User_Info::execute()
 
   RESTArgs::get_bool(s, "stats", false, &fetch_stats);
 
+  RESTArgs::get_bool(s, "sync", false, &sync_stats);
+
```

Comment:
`--sync-stats` seems already exists in rgw_admin. So could you please clarify how do you use the command now ?

The tracker shows: `$ radosgw-admin user stats --uid=<uid> --sync-stats`

Reply:
If you call this command without --sync-stats , the stats may be stale.  I noticed this when running some of the integration tests on an admin api client library I wrote.  i.e., I could copy a file to a bucket, but stats would return with all zeros.  All this PR would do is expose that same functionality via the admin API.

Reply:
Looks good. But I would request a review from @yehudasa too.